cmd/bbolt/main.go

```diff
@@ -60,12 +60,26 @@ func main() {
 	m := NewMain()
 	if err := m.Run(os.Args[1:]...); err == ErrUsage {
 		os.Exit(2)
+	} else if err == ErrUnknownCommand {
+		execute()
 	} else if err != nil {
 		fmt.Println(err.Error())
 		os.Exit(1)
 	}
 }
 
+func execute() {
+	rootCmd := NewRootCommand()
+	if err := rootCmd.Execute(); err != nil {
+		if rootCmd.SilenceUsage {
```

Comment:
What is the purpose of checking this field? From the documentation I understand it as it's a value to be used by Cobra library. It doesn't make sense to me the check this value here? Am I missing something?
https://pkg.go.dev/github.com/spf13/cobra#Command

Reply:
It prevents from printing duplicated error message. Please refer to https://github.com/etcd-io/etcd/pull/14173

Reply:
The commit you have referred is checking `SilenceErrors`, not `SilenceUsage`.

It's not a big deal though. All these can be refactored after migrating existing commands to Cobra.

Reply:
Good catch! Just changed it to `SilenceErrors`. Confirmed that it works,

```
$ ./bbolt surgery clear-page-elements ./local-kv.db --output ./new1.db --pageId 29 --from 5 --to 3
Error: clear-page-element command failed: the start index (5) is bigger than the end index (3)
Usage:
  bbolt surgery clear-page-elements [options] [flags]

Flags:
      --from int        start element index (included) to clear, starting from 0
  -h, --help            help for clear-page-elements
      --output string   path to the target db file
      --pageId uint     page id
      --to int          end element index (excluded) to clear, starting from 0, -1 means to the end of page
```

Reply:
```
If we always print the error message, it will print duplicated error message, 
$ ./bbolt surgery clear-page-elements ./local-kv.db --output ./new1.db --pageId 29 --from 5 --to 3
Error: clear-page-element command failed: the start index (5) is bigger than the end index (3)
Usage:
  bbolt surgery clear-page-elements [options] [flags]

Flags:
      --from int        start element index (included) to clear, starting from 0
  -h, --help            help for clear-page-elements
      --output string   path to the target db file
      --pageId uint     page id
      --to int          end element index (excluded) to clear, starting from 0, -1 means to the end of page

Error: clear-page-element command failed: the start index (5) is bigger than the end index (3)
exit status 1
```